packages/ui/cypress/utils/clickOnConnect.ts

```diff
@@ -0,0 +1,11 @@
+import { landingPage } from '../support/page-objects/landingPage'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+
+export const clickOnConnect = () => {
+  cy.clock()
+  topMenuItems.connectButton().click()
+  landingPage.accountsLoader().should('contain', 'Loading accounts')
+  cy.tick(500)
```

Comment:
As I understood from the documentation, it's a method to mock the time. I didn't find any setInterval in our implementation. Isn't it just pending Promise, or am I mistaken?

Reply:
Nope there's a settimeout to go around a bug https://github.com/ChainSafe/Multix/blob/2a1f23bf2b9e5803a0fe7e3795e9c0f1c5579ed1/packages/ui/src/contexts/AccountsContext.tsx#L135-L138